pages/cart.js

```diff
@@ -0,0 +1,232 @@
+import { useContext, useEffect, useState } from "react";
+import styled from "styled-components";
+import { toast } from "react-toastify";
+import { FaTimes } from "react-icons/fa";
+
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import { Divider, Button } from "../styles/UIKit";
+import useApi from "../utils/useApi";
+import CartConstants from "../config/CartConfig";
+import AuthContext from "../contexts/AuthContext";
+
+const Cart = () => {
+  const api = useApi();
+
+  const [cartData, setCartData] = useState({});
+  const { cart, updateCart } = useContext(AuthContext);
+
+  const fetchData = () => {
```

Comment:
this is not required, the cart object already has the required data

Reply:
taken care